util/topgen.py

```diff
@@ -1193,8 +1192,11 @@ def render_template(template_path: str, rendered_path: Path,
         # twice:
         # - Once under out_path/sw/autogen
         # - Once under hw/top_{topname}/sw/autogen
-        for path in [out_path.resolve(),
-                     (SRCTREE_TOP / 'hw/top_{}/'.format(topname)).resolve()]:
+        root_path = [out_path.resolve(), SRCTREE_TOP]
+        for idx, path in enumerate([
+                out_path.resolve(),
+            (SRCTREE_TOP / 'hw/top_{}/'.format(topname)).resolve()
+        ]):
```

Comment:
Nit: Something really weird happened to the formatting here. If you're using an auto-formatter, maybe pull the list of paths into a separate variable?

Reply:
Done. Please take a look